pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -330,6 +330,7 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 			}
 			return
 		}
+		// TODO: move in updateMetricsOnAllocate
```

Comment:
What is the reason for postponing this? Should we handle this in this PR itself?

Reply:
had to serialize quite some PRs in this area, not sure which one would be the last so can carry this cleanup

Reply:
...AND there 's a torny issue I don't have a solution yet. In theory, the code portion which owns alignment information is the topology object; hence, we should report metrics based on that. Yet, for historical reasons, we make logic on the `options` object. I don't know how to properly cleanup this yet.